app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -35,6 +36,10 @@ def show
 
       private
 
+      def capture_secondary_id_enabled
+        IdentityConfig.store.in_person_capture_secondary_id_enabled
```

Comment:
```suggestion
        !!current_user.establishing_in_person_enrollment&.capture_secondary_id_enabled
```
We should be looking at the enrollment rather than the feature flag, as we want to make sure individual users' experience is consistent (e.g. if the flag was on when they started the IPP flow, they should continue to see DAV pages even if we turn the flag off halfway through)

We could maybe set this value in `VerifyInfoConcern` as it's being used there in the shared `update` method.

Reply:
Resolved with commit 70e9b7685d6f74fd9874609f0b0e7a98d397cab9

Reply:
yep, that's exactly what I meant!